src/event/stream.rs

```diff
@@ -29,27 +30,36 @@ use super::{
 ///
 /// Check the [examples](https://github.com/crossterm-rs/crossterm/tree/master/examples) folder to see how to use
 /// it (`event-stream-*`).
-#[derive(Default)]
```

Comment:
Why do you remove default?

Reply:
Had a version where `EventStream` was constructed differently. Fixed!

Reply:
Ah, actually. Construction is no longer fallible, so we can't provide a `Default` impl :(